packages/shared/src/components/profile/ReadingStreaksWidget.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement } from 'react';
+import { ActivityContainer, ActivitySectionHeader } from './ActivitySection';
+import { ReadingStreakIcon } from '../icons';
+import { useReadingStreak } from '../../hooks/streaks';
+
+interface StreakTagProps {
+  streak: number;
+  title: string;
+  isLoading: boolean;
+}
+const StreakTag = ({ streak, title, isLoading }: StreakTagProps) => {
+  return (
+    <div className="relative flex h-[4.25rem] w-[10.125rem] flex-col rounded-10 border border-background-subtle p-3">
```

Comment:
Why do we need the weird h/w here?

Reply:
height we don't need actually, width we need for it to be as in design replaced with w-40